<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Игрок 1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Загадайте слово</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>221</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>220</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Выбрать</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Осталось неверных попыток: N</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Игрок 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>test</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>291</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Выберите букву</string>
     </property>
     <widget class="QPushButton" name="pushButton_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>а</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>б</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>в</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>г</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>д</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>е</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>ё</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>ж</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>з</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>и</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>й</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>к</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>л</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>м</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_15">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>н</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_16">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>о</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_17">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>п</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_18">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>р</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_19">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>с</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_20">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>т</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_21">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>у</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_22">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>ф</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_23">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>х</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_24">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>ц</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_25">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>50</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>ч</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_26">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>ш</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_27">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>щ</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_28">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>ъ</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_29">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>ы</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_30">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>ь</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_31">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>э</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_32">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>ю</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_33">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>21</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>я</string>
      </property>
     </widget>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>250</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>6</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Угадайте слово</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Who won?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_back">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Реванш</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
